iceoryx_posh/include/iceoryx_posh/internal/roudi/introspection/port_introspection.inl

```diff
@@ -195,38 +204,39 @@ bool PortIntrospection<SenderPort, ReceiverPort>::PortData::updateConnectionStat
     return true;
 }
 
-template <typename SenderPort, typename ReceiverPort>
-bool PortIntrospection<SenderPort, ReceiverPort>::PortData::addSender(typename SenderPort::MemberType_t* port,
-                                                                      const ProcessName_t& name,
-                                                                      const capro::ServiceDescription& service,
-                                                                      const RunnableName_t& runnable)
+template <typename PublisherPort, typename SubscriberPort>
+bool PortIntrospection<PublisherPort, SubscriberPort>::PortData::addPublisher(
+    typename PublisherPort::MemberType_t* port,
+    const ProcessName_t& name,
+    const capro::ServiceDescription& service,
+    const RunnableName_t& runnable)
 {
     std::lock_guard<std::mutex> lock(m_mutex);
 
-    auto iter = m_senderMap.find(service);
-    if (iter != m_senderMap.end())
+    auto iter = m_publisherMap.find(service);
+    if (iter != m_publisherMap.end())
         return false;
 
-    auto index = m_senderContainer.add(SenderInfo(port, name, service, runnable));
+    auto index = m_publisherContainer.add(PublisherInfo(port, name, service, runnable));
     if (index < 0)
```

Comment:
Braces are making @elBoberido happy!

Here are some extra braces for your happiness {}{}{}{}{}{}{}{}{}{}{}{}